src/sentry/workflow_engine/migration_helpers/alert_rule.py

```diff
@@ -529,3 +583,45 @@ def dual_delete_migrated_alert_rule(
         RegionScheduledDeletion.schedule(instance=data_source, days=0, actor=user)
 
     return
+
+
+def dual_delete_migrated_alert_rule_trigger(
+    alert_rule_trigger: AlertRuleTrigger,
+    user: RpcUser | None = None,
+) -> None:
+    priority = (
+        DetectorPriorityLevel.MEDIUM
+        if alert_rule_trigger.label == "warning"
+        else DetectorPriorityLevel.HIGH
+    )
+    detector_trigger = get_detector_trigger(alert_rule_trigger, priority)
+    if detector_trigger is None:
+        return None
+
+    action_filter = get_action_filter(alert_rule_trigger, priority)
+    if action_filter is None:
+        return None
+
+    detector_trigger.delete()
+    action_filter.delete()
+    logger.info("Dual deleted alert_rule_trigger", extra={"alert_rule_trigger": alert_rule_trigger})
```

Comment:
I don't think you want to log the entire `alert_rule_trigger` object (maybe the id?), and since it's about to be deleted as well I'm not sure how useful the info would be 🤔